src/components/HeaderThemeContainer/HeaderThemeContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ThemeToggle } from '../ThemeToggle/ThemeToggle';
+
+import './HeaderThemeContainer.scss';
+
+export function HeaderThemeContainer() {
```

Comment:
Tutaj dopieściłbym 2 rzeczy - nazwa komponentu i jego odpowiedzialność. Ustawianie theme powinno się znaleźć w ThemeToggle - to jedyny element z tego komponentu, który za to tak naprawdę odpowiada, reszta to po prostu jakieś semantyczne znaczniki. Nazwa w takim przypadku też zostanie uproszczona i ten 'theme' powinien z niej zniknąć